api/internal/core/entity/format.go

```diff
@@ -46,13 +50,35 @@ func NodesFormat(obj interface{}) interface{} {
 			nodes = append(nodes, node)
 		}
 		return nodes
-	}
+	case map[string]interface{}:
+		log.Infof("nodes type: %v", objType)
+		var strArr []string
+		value := obj.(map[string]interface{})
+		for key, val := range value {
+			node := &Node{}
+			strArr = strings.Split(key, ":")
```

Comment:
Key splitting can be extracted as a function, it's used multiple times.

Reply:
updated.